exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -121,24 +134,40 @@ class OTLPExporterMixin(
 
     Args:
         endpoint: OpenTelemetry Collector receiver endpoint
+        insecure: Connection type
         credentials: ChannelCredentials object for server authentication
         metadata: Metadata to send when exporting
     """
 
     def __init__(
         self,
-        endpoint: str = "localhost:55680",
+        endpoint: Optional[str] = None,
+        insecure: Optional[bool] = None,
         credentials: ChannelCredentials = None,
         metadata: Optional[Tuple[Any]] = None,
     ):
         super().__init__()
 
-        self._metadata = metadata
+        endpoint = (
+            endpoint
+            or Configuration().EXPORTER_OTLP_ENDPOINT
+            or "localhost:55680"
+        )
+
+        if insecure is None:
+            insecure = Configuration().EXPORTER_OTLP_INSECURE
+        if insecure is None:
+            insecure = False
+
+        self._metadata = metadata or Configuration().EXPORTER_OTLP_HEADERS
```

Comment:
Should this be called `self._headers`?

Reply:
This was introduced probably because [grpc client arg](https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-proto/src/opentelemetry/proto/collector/trace/v1/trace_service_pb2_grpc.py#L70) is named metadata too. Do you suggest we rename on exporter side to headers?

Reply:
Yes. It matches what the env variable name is.